packages/backend/src/lib/backends/github.sync.ts

```diff
@@ -93,17 +96,79 @@ export async function getInsightFromRepository(owner: string, repo: string): Pro
   const repository = await getRepository(owner, repo);
   logger.debug(JSON.stringify(repository, null, 2));
 
+  const insight: IndexedInsight = {
+    itemType: ItemType.INSIGHT,
+    namespace: repository.owner.login,
+    name: repository.name,
+    fullName: `${repository.owner.login}/${repository.name}`,
+    description: repository.description ?? '',
+    createdAt: repository.createdAt!,
+    updatedAt: repository.updatedAt!,
+    syncedAt: new Date().toISOString(),
+    stars: repository.stargazers!.totalCount,
+    forks: repository.forkCount!,
+    tags: repository.repositoryTopics!.edges.map(({ node }: any) => node.topic.name),
+    contributors: [],
+    repository: {
+      externalId: repository.id,
+      externalFullName: repository.nameWithOwner!,
+      externalName: repository.name,
+      type: RepositoryType.GITHUB,
+      defaultBranch: repository.defaultBranchRef?.name || 'master',
+      url: repository.url,
+      cloneUrl: repository.cloneUrl,
+      owner: {
+        type: repository.owner.__typename! as string as PersonType,
+        login: repository.owner.login,
+        externalId: repository.owner.id,
+        avatarUrl: repository.owner.avatarUrl!
+      },
+      isMissing: false,
+      isArchived: repository.isArchived
+    }
+  };
+
+  logger.debug(JSON.stringify(insight, null, 2));
+
+  return insight;
+}
+
+async function getInsightContributors(insight: IndexedInsight, yaml: InsightYaml): Promise<IndexedInsightUser[]> {
+  const userServices = new UserService(new ActivityService());
+
+  // If authors is manually specified in the YAML, use that instead of the GitHub API
+  if (yaml.authors && yaml.authors.length > 0) {
+    const contributors = await pMap(yaml.authors, async (author) => {
+      const user = await userServices.getUserByEmail(author);
+
+      return user === null
+        ? {
+            userName: author,
+            displayName: author,
+            email: author
+          }
+        : {
+            userId: user.userId,
+            userName: user.userName,
+            email: user.email,
+            displayName: user.displayName,
+            avatar: user.avatar
+          };
```

Comment:
If we have an IEX user with the same email, we pull in their details.  If not, just the email will display in the UI.

Reply:
Nice!